packages/server/src/organization/organization.controller.ts

```diff
@@ -874,13 +875,14 @@ export class OrganizationController {
     if (organization.logoUrl) {
       fs.unlink(
         process.env.UPLOAD_LOCATION + '/' + organization.logoUrl,
-        (err) => {
-          if (err) {
-            const errMessage =
+        (e) => {
+          if (e) {
+            console.error(
               'Error deleting previous picture at : ' +
-              organization.logoUrl +
-              'the previous image was at an invalid location?';
-            console.error(errMessage);
+                organization.bannerUrl +
```

Comment:
should this be logoUrl instead of bannerUrl?

Reply:
Whoops, I copied and pasted and forgot to change that. Thanks!